src/compiler/diagnosticMessages.json

```diff
@@ -1498,6 +1498,10 @@
         "category": "Error",
         "code": 2207
     },
+    "This type parameter likely needs an `extends object` constraint.": {

```

Comment:
I'd prefer 'probably' instead of 'likely' because it's a more common word. Up to you.